services/open-vsx/open-vsx-downloads.tester.js

```diff
@@ -0,0 +1,64 @@
+'use strict'
+
+const t = (module.exports = require('../tester').createServiceTester())
+const { withRegex, isMetric } = require('../test-validators')
+
+const isNotFound = withRegex(/^extension not found$/)
```

Comment:
I was planning on directly returning the error message from the API and use that for comparison, but later on I realize there's no point in doing so, since all the error is probably related with not found extension. 
So yes a literal raw match is definetly better here. I will update it.